src/Instrumentation/PDO/README.md

```diff
@@ -33,4 +33,12 @@ otel.instrumentation.pdo.distribute_statement_to_linked_spans = true
 or environment variable:
 ```shell
 OTEL_PHP_INSTRUMENTATION_PDO_DISTRIBUTE_STATEMENT_TO_LINKED_SPANS=true
+```
+
+## Database Context Propagation
+
+Enable context propagation for database queries by installing the following packages:
+```shell
+composer require open-telemetry/opentelemetry-context-propagator
+composer require open-telemetry/opentelemetry-sqlcommenter
```

Comment:
Right now it would be enabled for both `postgresql` or `mysql` under PDO, can we add a config to enable only specific driver?

Reply:
@brettmc / @open-telemetry/php-maintainers  Do you think it is a good idea to create a config to enable only specific driver?
e.g.
`OTEL_PHP_SQLCOMMENTER_OPT_IN_DATABASE` where default value = mysql and valid values are { mysql, postgresql }

Reply:
For more information, the hardcoding logic https://github.com/jerrytfleung/opentelemetry-php-contrib/blob/service_name_propagator/src/Instrumentation/PDO/src/PDOInstrumentation.php#L135-L137 was approved in #406

Reply:
I'd be happy to wait for somebody to ask for it, this is a `0.1` release